access.md

```diff
@@ -1,31 +1,42 @@
 ## Accessing a virtual machine
 
-Once a virtual machine got started you are able to connect to the consoles it exposes. Usually there are two types of consoles:
+Once a virtual machine got started you are able to connect to the consoles it
+exposes. Usually there are two types of consoles:
 
 * Serial Console
 * Graphical Console
 
-> Note: You need to have `virtctl` [installed](installation.md) to gain access to the VirtualMachine.
+> Note: You need to have `virtctl` [installed](installation.md) to gain access
+> to the VirtualMachine.
 
 ### Accessing the serial console
 
-The serial console of a virtual machine can be accessed by using the `console` command:
+The serial console of a virtual machine can be accessed by using the `console`
+command:
 
 ```bash
 $ virtctl console -s http://mycluster:8184 testvm
 ```
 
 ### Accessing the graphical console
 
-Accessing the graphical console of a virtual machine is usually done through SPICE, which requires a SPICE client like `remote-viewer`. Once the tool is installed you can access the graphical console using:
+Accessing the graphical console of a virtual machine is usually done through
+SPICE, which requires a SPICE client like `remote-viewer`. Once the tool is
+installed you can access the graphical console using:
 
 ```bash
 $ virtctl spice -s http://mycluster:8184 testvm
 ```
 
+Optional, `--proxy` (e.g. `http://192.168.200.2:1234`) can be used. This is
+useful, when you are outside of the cluster and no or a non-reachable proxy is
```

Comment:
The intent of this sentence isn't entirely clear. I *think* is should have said:

This is useful when you are outside of the cluster or a non-reachable proxy is provided by the system.

(also note I removed the comma after "this is useful").